.github/workflows/mend.yaml

```diff
@@ -6,19 +7,19 @@ on:
       - main
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
     steps:
     - name: connect_twingate
       uses: twingate/github-action@v1
       with:
         service-key: ${{ secrets.TWINGATE_PUBLIC_REPO_KEY }}
```

Comment:
Do we have this? Below is also a `MEND_API_KEY` and it looks like this whole workflow just fails. Should we drop it?